jsonschema-core.xml

```diff
@@ -2953,52 +3076,127 @@ https://example.com/schemas/common#/$defs/count/minimum
                         Because no errors or annotations are returned with this format, it is
                         RECOMMENDED that implementations use short-circuiting logic to return
                         failure or success as soon as the outcome can be determined.  For example,
-                        if an "anyOf" keyword contains five sub-schemas, and the second one
+                        if an "anyOf" keyword contains five subschemas, and the second one
                         passes, there is no need to check the other three.  The logic can simply
                         return with success.
                     </t>
                 </section>
 
                 <section title="Basic">
                     <t>
-                        The "Basic" structure is a flat list of output units.
+                        The "Basic" structure is a flat list of output units contained within a
+                        root output unit.
+                    </t>
+                    <t>
+                        The root output unit contains "valid" for the overall result and "nested"
+                        for the list of specific results.  All other information is explicitly
+                        omitted from the root output unit.  If the root schema produces errors or
+                        annotations, then the output node for the root MUST be present within the
+                        root output unit's "nested" list with those errors or annotations.
+                    </t>
+                    <t>
+                        Output units which do not contain errors or annotations SHOULD be excluded
+                        from this format, however implementations MAY choose to include them for
+                        completeness.
```

Comment:
Is this intended to allow implementations to do things like raise an exception?

Reply:
No, just to reduce the payload size.  If there are no errors or annotations, the output unit doesn't give much info (just locations and a validation result) and isn't very helpful.